tools/labelme/labelme.go

```diff
@@ -56,3 +85,18 @@ func RectAnnotationToShape(ann annotation.RectAnnotation, lineColor *[4]int, fil
 	s.Points = append(s.Points, [2]int{ann.X, ann.Y + ann.Height})
 	return s
 }
+
+func colorStringToIntArray(s string) [4]int {
```

Comment:
You have 4 ints, so you should name it “parseRGBAHexColor” which explains the data of the input.